src/sections/CareersHero/index.js

```diff
@@ -1,47 +1,116 @@
-import React from "react";
-
+import React, { useLayoutEffect, useRef } from "react";
 import Section from "root/components/Section";
 import Navbar from "root/components/Navbar";
 import Typography from "root/components/Typography";
 import Wave from "root/assets/images/wave-careers-hero.inline.svg";
+import useScroll from "root/hooks/useScroll";
 
 import "./index.css";
 
-const CareersHero = () => (
-  <div styleName="root">
-    <Navbar currentPage="careers" theme="primary" />
-    <Section verticalSpacing={false}>
-      <div styleName="title-desktop">
-        <div styleName="first-text">
+function CareersHero() {
+  const scrollPosition = useScroll();
+  const textRef = useRef();
+  const textContainerRef = useRef();
+
+  const textPath = textRef ? textRef.current : null;
+  const textContainer = textContainerRef ? textContainerRef.current : null;
+
+  useLayoutEffect(() => {
+    const path = textPath
+      ? document.querySelector(textPath.getAttribute("href"))
+      : null;
+
+    if (path) {
+      const pathLength = path.getTotalLength();
+      const initialPosition = 445;
+
+      const rect = textContainer.getBoundingClientRect();
+      const scrollPercent = rect.y / 600;
+
+      updateTextPathOffset(scrollPercent * pathLength, initialPosition);
```

Comment:
if initialPosition is always 445 we don't need to create the variable and pass as parameter. It should exist inside `updateTextPathOffset`

Reply:
True 👍